steps/expression_processor.go

```diff
@@ -21,7 +21,52 @@ func RegisterExpression(b reg.ProcessorRegistry) {
 			return add_expression(p, params, false)
 		},
 		"Execute the given expression on every sample").
-		Required("expr", reg.String())
+		Required("expr", reg.String(), "Allows arithmetic and boolean operations. Can also perform them on sample fields "+
+			"(e.g. \"field1\" [+,-,*,/] \"field2\").",
+			"Following additional functions are implemented: ",
+			"tag(string) string: Access tag by tag key (string). Returns tag string or empty string if key does not exist.",
+			"",
+			"has_tag(string) bool: Check existence of tag key.",
+			"",
+			"set_tag(string, string) bitflow.SampleAndHeader: Adds or replaces a tag at the current sample and returns the result.",
+			"First argument is key, second is the tag value.",
+			"",
+			"timestamp() float64: Returns the timestamp of the current sample.",
+			"",
+			"now() float64: Returns the current local system time.",
+			"",
+			"num() float64: Returns the current number of processed samples.",
+			"",
+			"str(_) string: Converts an arbitrary argument to string.",
+			"",
+			"strToFloat(string) float64: Converts a string argument to float64.",
+			"",
+			"date_str(float64) string: Converts the timestamp argument to a string representation.",
+			"",
+			"set_timestamp(float64) bitflow.SampleAndHeader: Sets the timestamp of the current sample and returns the result.",
+			"",
+			"floor(float64) float64: Applies the floor operation on the argument.",
+			"",
+			"set(string, float64, optional: bitflow.SampleAndHeader) bitflow.SampleAndHeader:",
+			" Sets or replaces the value (2nd argument) of the sample field (1st argument).",
+			"Third argument is optional. If set, the setting is applied on the passed sample.",
+			"Otherwise it is applied on the current sample.",
+			"",
+			"get_sample_and_header(): bitflow.SampleAndHeader: Returns the current sample.",
+			"",
+			"Note that arithmetic, boolean and function expressions can be combines as long as the arguments and return types match.",
+			"Some examples: ",
+			"'expr'='set_tag(\"my_system_time\", str(now()))'",
+			"'expr'='set_timestamp(now())'",
+			"'expr'='set(\"field3\", \"field1\" + \"field2\")'",
```

Comment:
I think this example is not correct, because the second parmeter should not be a string, but rather field values, which are written without string quotes

Reply:
Same applies for the examples below that want to use field values

Reply:
Done